tests/oidc.yml

```diff
@@ -26,6 +25,3 @@ parameters:
         subjects:
           - kind: Group
             name: admin
-
-    syn:
-      registration_url: https://syn.example.com/steward/install.json?token=w84kxjbhf
```

Comment:
Does this still need to be removed? Afaict we're keeping this functionality now?

Reply:
You're right. I forgot to re-add this. Fixed.